spec/features/login_signup/login_spec.rb

```diff
@@ -18,31 +18,5 @@
     expect(page).to have_link('Sign Up')
   end
 
-  context 'Form Submission' do
-    scenario 'Submit form without email and password' do
-      click_button 'Log In'
-      expect(page).to have_content 'Invalid Email or password.'
-    end
-
-    scenario 'Submit form with incorrect email and password' do
-      within 'form' do
-        fill_in 'Email', with: 'newmail@gmail.com'
-        fill_in 'Password', with: '123456'
-      end
-      click_button 'Log In'
-      expect(page).to have_content 'Invalid Email or password.'
-    end
-
-    scenario 'Submit form with correct email and password' do
-      @user = User.create(name: 'Dammy', email: 'admin@gmail.com', password: 'password')
-      within 'form' do
-        fill_in 'Email', with: @user.email
-        fill_in 'Password', with: @user.password
-      end
-      click_button 'Log In'
-      expect(page).to have_current_path(authenticated_root_path)
-      expect(page).to have_content 'Signed in successfully.'
-    end
-  end
   # rubocop:enable Metrics/BlockLength
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227706806-e6fc9187-fbdb-4a63-89ce-d976e2fa352b.png)

**[OPTIONAL]**

- [ ] It would be great if you make your commit messages capitalize to make your pull request professional :+1: